book/src/usage.md

```diff
@@ -148,4 +148,4 @@ clippy-driver --edition 2018 -Cpanic=abort foo.rs
 > that are not optimized as expected, for example.
 
 [Installation]: installation.md
-[CI]: continuous_integration/index.md
+[CI]: continuous_integration/README.md
```

Comment:
Looks like this change caused a new issue: https://github.com/rust-lang/rust/pull/110003#issuecomment-1500249457

Reply:
I have no idea how to fix it then.
`index.html` didn't work (even though it's the URL from the book leading to that section), and `README.md` also doesn't work, even though it [appears like that on `SUMMARY.md`](https://github.com/rust-lang/rust-clippy/blob/015fb8a9e33fccf8aa1e53a33f187940a845016f/book/src/SUMMARY.md?plain=1#L10).

Even then, all link's were checked with the `mdbook-linkcheck` crate before committing them. Could this be a bug in CI?

Reply:
https://github.com/rust-lang/rust-clippy/pull/10630